app/models/forms/bulk_upload_lettings/upload_your_file.rb

```diff
@@ -14,6 +21,49 @@ def view_path
       def back_path
         bulk_upload_lettings_log_path(id: "prepare-your-file", form: { year: })
       end
+
+      def year_combo
+        "#{year}/#{year + 1 - 2000}"
+      end
```

Comment:
Would it make sense to extract this somewhere? We're also using it in app/models/forms/bulk_upload_lettings/checking_file.rb

Reply:
Personally I think this level of duplication is fine. I don't really see this code changing much. If there were many more pages being added I think I would consider a base class or maybe some kind of decorator thing. I don't think we stand to gain much if we were to DRY this up at this moment in time